### DISCUSSION TITLE
Query

### DISCUSSION THREAD

**Discussion by shinchancode** [MakeContributions/DSA#436] [Q&A]
I m new to contributions and pull request. I have a query related to this "Resource not accessible by integration". Can anyone help me to deal with this error

**Comment by ming-tsai** [comment]
> I m new to contributions and pull request. I have a query related to this "Resource not accessible by integration". Can anyone help me to deal with this error

Could you send me the error link?

**Reply by shinchancode** [reply]
Yes,
https://github.com/MakeContributions/DSA/pull/434/checks?check_run_id=3455919686

**Comment by ming-tsai** [comment]
Don't worry about the `label` action, I have disabled it

**Comment by Emanuel181** [comment]
Hello, I made a pull request about sieve of eratosthenes a month ago, when the review will be done ?

**Reply by ming-tsai** [reply]
Sorry about that, due to this is an open source repo, so it is depende the reviewer is busy or not, by the way I have assigned another reviewer.

**Reply by Emanuel181** [reply]
Hello, I have opened another pull request because reviewer said that my algorithm already exist in C++ so I made it in C. Thank you !
